python_modules/libraries/dagster-dbt/dagster_dbt_tests/conftest.py

```diff
@@ -57,7 +57,9 @@ def disable_openblas_threading_affinity_fixture() -> None:
 def _create_dbt_invocation(project_dir: Path, build_project: bool = False) -> DbtCliInvocation:
     dbt = DbtCliResource(project_dir=os.fspath(project_dir), global_config_flags=["--quiet"])
 
-    dbt.cli(["deps"]).wait()
+    if not project_dir.joinpath("dbt_packages").exists():
+        dbt.cli(["deps"]).wait()
```

Comment:
theres still a theoretical race here with multiple process clearing the if block before the paths get written, not sure its worth fretting about since its a smaller window 

could try catch the specific exception to degrade gracefully

Reply:
ah yeah, we should just put `raise_on_error=False` here

Reply:
how likely is it that this fails for a different reason? is there still something in the logs? 

just considering if this would make debugging that case hard since we're hiding it

Reply:
Should be written out to stdout logs, but we just won't raise an exception